package.json

```diff
@@ -11,7 +11,7 @@
     "lint:fix": "eslint --fix",
     "test": "npm run test:unit && npm run test:typescript",
     "test:typescript": "tsd",
-    "test:unit": "tap",
+    "test:unit": "node --test --experimental-test-module-mocks",
```

Comment:
can you add `c8` for code coverage

Reply:
I had to configure also the pattern to avoid issues with node 20